packages/react-draggable-dialog/package.json

```diff
@@ -2,11 +2,16 @@
   "name": "@fluentui-contrib/react-draggable-dialog",
   "version": "0.0.1",
   "private": true,
+  "dependencies": {
+    "@dnd-kit/core": "^6.1.0",
+    "@dnd-kit/utilities": "^3.2.2",
+    "@dnd-kit/modifiers": "^7.0.0"
```

Comment:
<img width="822" alt="image" src="https://github.com/microsoft/fluentui-contrib/assets/14183168/7fadab20-8b0d-4f0d-a898-8410999a6589">


FYI: I haven't seen that bits of dnd-kit are used, but in some scenario it has extremely poor performance, https://github.com/clauderic/dnd-kit/issues/1194#issuecomment-1696704815